src/entity/effect/VanillaEffects.php

```diff
@@ -68,6 +69,7 @@ protected static function setup() : void{
 		//TODO: bad_omen
 		self::register("blindness", new Effect(KnownTranslationFactory::potion_blindness(), new Color(0x1f, 0x1f, 0x23), true));
 		self::register("conduit_power", new Effect(KnownTranslationFactory::potion_conduitPower(), new Color(0x1d, 0xc2, 0xd1)));
+		self::register("darkness", new Effect(KnownTranslationFactory::effect_darkness(), new Color(0x29, 0x27, 0x21), true));
```

Comment:
Darkness shouldn't have bubbles. You need to pass the final constructor parameter.